web_external/views/map/MapPanel.js

```diff
@@ -130,13 +138,25 @@ const MapPanel = Panel.extend({
                     ? mapSettings.basemap_args
                     : {});
             this.uiLayer = this.map.createLayer('ui');
-            this.uiLayer.createWidget('slider', { position: { right: 20, bottom: 30 } });
+            this.uiLayer2 = this.map.createLayer('ui');
```

Comment:
This would be clearer with a more explicit name (`controlLayer` perhaps?) and a comment saying that widgets on this layer won't be included in the screenshot.